src/Directory.Build.props

```diff
@@ -12,22 +12,19 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Akka Persistence journal and snapshot store backed by MongoDB database.</Description>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <LangVersion>latest</LangVersion>
```

Comment:
Most projects seem to be using latest. Can change this to what ever is appropriate.